sdk/textanalytics/Azure.AI.TextAnalytics/tests/samples/Sample7_AnalyzeHealthcareEntitiesBatchConvenienceAsync.cs

```diff
@@ -23,6 +23,7 @@ public async Task Sample7_AnalyzeHealthcareEntitiesBatchConvenienceAsync()
             var client = new TextAnalyticsClient(new Uri(endpoint), new AzureKeyCredential(apiKey));
 
             #region Snippet:TextAnalyticsSampleHealthcareBatchConvenienceAsync
```

Comment:
Is this one used anywhere? if not, we can delete the region and the end region for it